lib/writer2.c

```diff
@@ -292,6 +292,9 @@ ktxTexture2_SetImageFromStdioStream(ktxTexture2* This, ktx_uint32_t level,
  * Level, layer, faceSlice rather than offset are specified to enable some
  * validation.
  *
+ * @warning Client applications are responsible for freeing the original copy of
+ *          image memory referred to by \p src.
```

Comment:
These requests apply both to here and `writer1.c`.

Please change `\p` to `@p` and change the text to

"The caller is responsible for freeing the original image memory referred to by @p src."

Both changes are for consistency with the rest of the documentation.

I think this should be `@note` not `@warning` because the `const` already implies that the function will not change the pointed-at content and therefore the caller is responsible for freeing it.

Reply:
Thank you for the review. I've updated the docs with your suggested changes.